src/main/java/org/testng/TestRunner.java

```diff
@@ -1483,7 +1461,7 @@ public static int getVerbose() {
     return m_verbose;
   }
 
-  public void setVerbose(int n) {
+  public static void setVerbose(int n) {
```

Comment:
Maybe, but I don't understand the goal to refactor something and keeping it wrong.